integration-test/pubspec.yaml

```diff
@@ -0,0 +1,27 @@
+name: project
+description: A sample flutter application for sentry-cli integration test.
+version: 1.1.0
+
+environment:
+  sdk: '>=2.18.0 <3.0.0'
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  sentry_dart_plugin:
+    path: ./../
+
+flutter:
+  uses-material-design: true
+
+sentry:
```

Comment:
I'd enable `upload_source_maps` and `include_native_sources` so we also test source maps and native sources.
You'll need a build for Flutter web as well.

Reply:
I have added a web target and also enables `include_native_sources`. I'm not sure though if any are "uploaded".

<img width="487" alt="Bildschirmfoto 2022-09-26 um 11 43 33" src="https://user-images.githubusercontent.com/3984453/192248959-827b7698-8230-4ebb-a237-1d74b5ea2e9f.png">

Reply:
For Web, its missing the `--source-maps` yet.
For sources, you will need to add a native lib in your Android example, such as https://github.com/getsentry/sentry-java/tree/main/sentry-samples/sentry-samples-android/src/main/cpp
Also requires the cmakelist file, only then it generates symbols with source code, so you can validate the params.

Reply:
Added a cpp sample library and call it from the android side. sentry-cli picked up the files and sends payloads to the fake server.

<img width="1198" alt="Bildschirmfoto 2022-09-27 um 10 47 31" src="https://user-images.githubusercontent.com/3984453/192484182-9b85628b-9015-4fa9-b4e6-4392c6ee1cfb.png">

<img width="671" alt="Bildschirmfoto 2022-09-27 um 10 43 28" src="https://user-images.githubusercontent.com/3984453/192484175-0fb26feb-1b85-45bf-97d3-a0b824d11b53.png">